cmd/libsnap-confine-private/snap.c

```diff
@@ -27,25 +27,34 @@
 #include "string-utils.h"
 #include "cleanup-funcs.h"
 
-bool verify_security_tag(const char *security_tag)
+bool verify_security_tag(const char *security_tag, const char *snap_name)
 {
 	// The executable name is of form:
```

Comment:
Can you move this to the documentation of the function in the header file please